users/users.spec.js

```diff
@@ -0,0 +1,33 @@
+const request = require('supertest');
+const db = require('../database/dbConfig');
+
+const Users = require('./users-model');
+
+describe('add()', () => {
+
+    beforeEach(async () => {
+        await db('users').truncate();
+    })
+
+    it('should find user by id', async () => {
+        await Users.add({id: 679, username: 'jeff', password: 'pass'});
+
+        const user = await Users.findById(679)
+        expect(user).toHaveLength(1)
+    })
+})
+
+describe('findById()', () => {
+
+    beforeEach(async () => {
+        await db('users').truncate();
+    })
+
+    it('should find user by id', async () => {
+        await Users.add({id: 679, username: 'jeff', password: 'pass'});
+
+        const user = await Users.findById(679)
+        expect(user).toHaveLength(1)
+    })
+})
```

Comment:
you might want to dabble with using mocking functions and some more advanced concepts in testing -- what we covered in this sprint was not nearly enough to say that you are proficient in TDD. Keep stretching yourself to include these on every code challenge you do.